lib/rust/ensogl/core/src/system/gpu/context/extension.rs

```diff
@@ -164,3 +166,53 @@ impl ExtDisjointTimerQueryWebgl2 {
         }
     }
 }
+
+
+
+// ========================
+// === WebglLoseContext ===
+// ========================
+
+/// Supports losing the WebGL Context.
```

Comment:
Please extend those docs a little. What does it means it supports? It's a wrapper for WEBGL_lose_context?

We actually we could just use lose_context method from web_sys: https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.WebglLoseContext.html#method.lose_context